### DISCUSSION TITLE
Device name is leaking?

### DISCUSSION THREAD

**Discussion by U-Gautam** [bromite/bromite#1456] [Q&A] [answered]
![Screenshot_2021-10-08-01-57-19-60_ed7821ed99ee442dec4ede205e7971d6_1](https://user-images.githubusercontent.com/92106554/136411581-9f51055f-bb59-4366-83fd-022c9e1f55f1.jpg)
if you goto browserleaks to test bromite, you can clearly see it leaks? the device name. This problem seems to be in brave, mulch as well. so i guess it is a chromium thing. It is very easy to fingerprint isn't it with so many android phones and everyone has different model. Firefox doesn't have this problem.

**Comment by csagan5** [comment, marked as answer]
It is enabled by default due to the client hints; will be disabled on next release.
